subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -47,6 +47,21 @@ For example, the use of deprecated APIs in Kotlin DSL will emit warnings each ti
 
 This is a potentially breaking change if you are consuming the console output of Gradle builds.
 
+==== Gradle caches may contain CACHEDIR.TAG files
+Gradle now emits a `CACHEDIR.TAG` file in some cache directories (see the release notes for details).
+
+This may cause these directories to no longer be searched or backed up by some tools. In order to disable
```

Comment:
This sounds a little indecisive: "may contain", "some cache directories", "may cause".  I understand it's the truth, but I'd consider re-titling the section to something more certain, that makes it sound slightly more positive.

Maybe: "Gradle properly tags cache dirs", and leave the qualifying statements to the body.

Reply:
I left the positive wording for the release notes, in the upgrading guides I think a more technical register is appropriate. I'll adjust it slightly.